### PR TITLE
Minor syzkaller code cleanup.

### PR DESCRIPTION
**Pull request by inferno-chromium**

### CODE REVIEW

File: src/python/bot/fuzzers/syzkaller/runner.py [hunk 1]
```diff
@@ -118,7 +120,7 @@ def fuzz(self,
           visited.add(unique_crash)
           log_lines = utils.read_data_from_file(
               os.path.join(subdir, file), eval_data=False)
-          fuzz_result.output = str(log_lines)
+          fuzz_logs += str(log_lines) + '\n'
```
Comment (inferno-chromium) [inline src/python/bot/fuzzers/syzkaller/runner.py:123]: fuzz_logs should have logs from all crashes. and engine.Crash is fine as-is with log_lines from each crash.

File: src/python/bot/fuzzers/syzkaller/engine.py [hunk 2]
```diff
@@ -122,9 +122,9 @@ def reproduce(self, target_path, input_path, arguments, max_time):
     Returns:
       A ReproduceResult.
     """
-    binary_path = self.prepare_binary_path()
+    binary_dir = self.prepare_binary_path()
```
Comment (oliverchang) [inline src/python/bot/fuzzers/syzkaller/engine.py:125]: nit: also rename prepare_binary_path ?

Comment (inferno-chromium) [reply src/python/bot/fuzzers/syzkaller/engine.py:125]: It actually does prepare binary path by fixing execute perms on the binary. returning binary dir is just a helper.

**Pull request merged**